releasetasks/templates/firefox/enUS.yml.tmpl

```diff
@@ -154,15 +154,11 @@
 
             env:
                 FILENAME_TEMPLATE: "{{ product }}-{{ partial_version }}-{{ version }}.en-US.{{ buildbot2ftp(platform) }}.partial.mar"
-                {% if signing_class == "dep-signing" %}
-                MOZ_DISABLE_MAR_CERT_VERIFICATION: true
-                SIGNING_CERT: "dep"
-                {% elif signing_class == "nightly-signing" %}
-                SIGNING_CERT: "nightly"
-                {% elif signing_class == "release-signing" %}
-                SIGNING_CERT: "release"
+                {% if moz_disable_mar_cert_verification is defined %}
```

Comment:
You probably want `if moz_disable_mar_cert_verification is defined and moz_disable_mar_cert_verification`. You don't want `moz_disable_mar_cert_verification` set to `False` disable the checks.